music.py

```diff
@@ -1,21 +1,19 @@
 import random
 import threading
-from utils import log
 from os import listdir
+
 from pydub import AudioSegment
 from pygame import mixer
-import config
 
+import config
+from utils import log
 
 class MusicPlayer:
     def __init__(self):
         mixer.init()
-        self.random_songs = self.find_random_songs()
-        self.random_stack = []
         self.ready_to_queue = False
         self.song_queue = []
         self.user_song_currently_playing = None
-
```

Comment:
It's okay to remove this newline, but if you feel that it helps break up the logical segments then you can leave it. I actually think it reads slightly better with the newline here.